### PR TITLE
Constants and params

### PR DESCRIPTION
**Pull request by ashcrow**
The dispatch method was getting to complex for it's own good. This
change pulls the parameter parsing out of dispatch to it's own method.

### CODE REVIEW

**Comment by ashcrow** [comment]
[(3) [multi-host-mgr] Enhance http server and base](https://trello.com/c/UQ6kgBDm/489-3-multi-host-mgr-enhance-http-server-and-base)

File: src/commissaire_http/constants.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+"""
+
+# JSONRPC error codes
+JSONRPC_ERRORS = {
+    'INVALID_JSON': -32700,
+    'INVALID_REQUEST': -32602,
+    'METHOD_NOT_FOUND': -32601,
+    'INVALID_PARAMETERS': -32602,
+    'INTERNAL_ERROR': -32603,
+}
```
Comment (mbarnes) [inline src/commissaire_http/constants.py]: These would make more sense in `commissaire.constants`.

Comment (ashcrow) [inline src/commissaire_http/constants.py]: Fair enough. I'll move them over there. Will update shortly.

Comment (ashcrow) [inline src/commissaire_http/constants.py]: Moved them to https://github.com/projectatomic/commissaire/pull/14. Will update this PR with a fixup.

**Comment by ashcrow** [comment]
Restarting build.

**Comment by ashcrow** [comment]
:arrow_up:

**Comment by ashcrow** [comment]
Squashed so I could cherry-pick and continue working while this is under review.

**Pull request merged**